lfs/ssh.go

```diff
@@ -22,7 +23,11 @@ func sshAuthenticate(endpoint Endpoint, operation, oid string) (sshAuthResponse,
 
 	tracerx.Printf("ssh: %s git-lfs-authenticate %s %s %s",
 		endpoint.SshUserAndHost, endpoint.SshPath, operation, oid)
-	cmd := exec.Command("ssh", endpoint.SshUserAndHost,
+	ssh := os.Getenv("GIT_SSH")
+	if len(ssh) == 0 {
+		ssh = "ssh"
+	}
```

Comment:
Ah you're right, I'm thinking of one of our internal projects :) I still think it's a good thing to do, especially for anything that's called multiple times. Two of those 4 matches are cached.